assets/src/edit-story/app/api/test/removeImagesFromPageTemplates.js

```diff
@@ -21,7 +21,7 @@ import getAllTemplates from '@web-stories-wp/templates';
 /**
  * Internal dependencies
  */
-import removeImagesFromPageLayouts from '../removeImagesFromPageLayouts';
+import removeImagesFromPageLayouts from '../removeImagesFromPageTemplates';
```

Comment:
```suggestion
import removeImagesFromPageTemplates from '../removeImagesFromPageTemplates';
```

Reply:
Oops, thank you! Looks like there are a few more places incorrectly using `...PageLayout` in constants/imports. _Replacing_